pkgs/os-specific/linux/libcgroup/default.nix

```diff
@@ -1,26 +1,21 @@
-{ lib, stdenv, fetchurl, fetchpatch, pam, yacc, flex }:
+{ lib, stdenv, fetchFromGitHub, pam, yacc, flex, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "libcgroup";
-  version = "0.41";
+  version = "0.42.2";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/libcg/${pname}-${version}.tar.bz2";
-    sha256 = "0lgvyq37gq84sk30sg18admxaj0j0p5dq3bl6g74a1ppgvf8pqz4";
+  src = fetchFromGitHub {
+    owner = pname;
```

Comment:
I am not sure if this is credible. There website does not link it.

Reply:
It's what the [Fedora project is currently using](https://src.fedoraproject.org/rpms/libcgroup/blob/17704306e5f5e3a42edd3254f414c72996a5b138/f/libcgroup.spec#_7), the [mailing list libcg-devel](https://sourceforge.net/p/libcg/mailman/libcg-devel/) seems to match what's in the Git repository on GitHub and there are messages about features specific to GitHub (see this thread for example: https://sourceforge.net/p/libcg/mailman/libcg-devel/thread/20201130171650.14825-3-tom.hromatka%40oracle.com/#msg37165452).

Reply:
FYI the move to GitHub is now mentioned on the SourceForge page.

https://sourceforge.net/projects/libcg/